src/styles/_no-feedback.scss

```diff
@@ -0,0 +1,30 @@
+.no-feedback {
+    max-width: 825px;
+    height: 600px;
+    border-radius: $border-radius;
+    background-color: $white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: $font-jost;
+
+    img {
+        margin-bottom: 55px;
+    }
+
+    span:first-of-type {
```

Comment:
This could be reduced to:
```
    span {
       color: $dark-gray;
          &:first-of-type {
            font-weight: 500;
            font-size: 24px;
            font-weight: 600;
            margin-bottom: 15px;
          }
          &:last-of-type {
            font-size: 16px;
            max-width: 410px;
            margin-bottom: 50px;
          }
    }
```

This way we keep the CSS scoped to the `span` and we are keeping the color the same across all spans